src/imprint.njk

```diff
@@ -8,9 +8,9 @@ title: Imprint
   <div class='imprint-box'>
     <h1 class="imprint-title">{{ title }}</h1>
     <p>
-      <strong>Responsible contact person:</strong> Ola Gasidlo-Braendel</p>
+      <strong>Responsible contact person:</strong> Ola Gasidlo‑Braendel</p>
     <p>
-      <strong>Email address:</strong> ola.gasidlo-braendel@accenture.com</p>
+      <strong>Email address:</strong> ola.gasidlo‑braendel@accenture.com</p>
```

Comment:
As said in Slack: We probably shouldn't change the email address here

Reply:
Yes, totally right. Changed it.